doc/source/dev/api-dev/array_api.rst

```diff
@@ -240,7 +240,7 @@ The following pytest markers are available:
 
 * ``array_api_compatible -> xp``: use a parametrisation to run a test on
   multiple array backends.
-* ``skip_xp_backends(*backends, reasons=None, np_only=False, cpu_only=False, exceptions=None)``:
+* ``skip_xp_backends(*backends, reason=None, np_only=False, cpu_only=False, exceptions=None)``:
   skip certain backends and/or devices.
   ``@pytest.mark.usefixtures("skip_xp_backends")`` must be used alongside this
   marker for the skipping to apply. See the fixture's docstring in ``scipy.conftest``
```

Comment:
Can you update this paragraph with the correct signature and point to the new docstrings. Also, could you include `xfail_xp_backends` too?

Reply:
Happy to see your preferred wording, as a commit or a diff for me to apply.